content/07.pathogenesis.md

```diff
@@ -334,33 +326,33 @@ Contact, droplet, and aerosol transmission are therefore all worth evaluating wh
 For example, the detection of SARS-CoV-2 viral particles in air samples taken from hospitals treating COVID-19 patients led to the concern that the virus could be spreading via aerosols [@doi:10.1038/s41586-020-2271-3].
-The stability of the virus both in aerosols and on a variety of surfaces appeared similar to that of SARS-CoV-1 [@doi:10.1056/NEJMc2004973], and fomite transmission could also play a role in transmission (e.g., [@doi:10.3201/eid2606.200412]).
+The stability of the virus both in aerosols and on a variety of surfaces appeared similar to that of SARS-CoV-1 [@doi:10.1056/NEJMc2004973], and fomite transmission could also play a role (e.g., [@doi:10.3201/eid2606.200412]).
 However, while the possibility of aerosol transmission seems plausible, the evidence suggests that droplet and contact transmission are the dominant mechanisms driving the spread of the virus [@doi:10.1001/jama.2020.12458], and the risk of fomite transmission under real-world conditions is likely to be substantially lower than the conditions used for experimental analyses [@doi:10/gg6br7].
 These mechanisms may differ in their relevance to different types of transmission events, such as transmission within households, nosocomial transmissions, and transmission in indoor versus outdoor spaces.
 
 #### Symptoms and Viral Spread
 
 Other aspects of pathogenesis are also important to understanding how the virus spreads, especially the relationship between symptoms, viral shedding, and contagiousness.
 Symptoms associated with reported cases of COVID-19 range from mild to severe [@url:https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html], but some individuals who contract COVID-19 remain asymptomatic throughout the duration of the illness [@doi:10.1007/s11427-020-1661-4].
-The incubation period, or time period between exposure and the onset of symptoms, has been estimated at five to eight days (4.91 and 7.54 in two different cities) [@doi:10.7554/eLife.57149; @doi:10.1164/rccm.202003-0524LE], and estimates suggest that viral shedding may begin long before the onset of symptoms (12.3 days with a 95% CI of 5.9 - 17.0) and peak around the onset of symptoms [@doi:10.1038/s41591-020-0869-5].
-As these trends became apparent, concerns arose due to the potential for individuals who did not or did not yet show symptoms to transmit the virus [@url:https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/how-covid-spreads.html].
+The incubation period, or time period between exposure and the onset of symptoms, has been estimated at five to eight days (4.91 and 7.54 in two different cities) [@doi:10.7554/eLife.57149; @doi:10.1164/rccm.202003-0524LE], and estimates suggest that viral shedding may begin long before the onset of symptoms (12.3 days with a 95% CI of 5.9 - 17.0) and peak around the onset of symptoms (95% CI, -0.9 - 0.9 days) [@doi:10.1038/s41591-020-0869-5].
```

Comment:
@SiminaB says:

> I think this is probably the median length – please check. I was looking into this a couple of weeks ago and there is a distribution of incubation times that has a long tail.

Reply:
Perhaps a related question. I think I asked about this already so I apologize for asking again. So The way I'm thinking about this is a little confusing. Say your day of symptom onset = 0. Then based on this description, your exposure day = - 5, but your viral shedding can start at day -12. How can you have viral shedding start before your exposure? Though perhaps I'm misunderstanding what they mean by exposure.

Reply:
@ajlee21 I think it's easier to think about it in the other direction -- if you're exposed to a virus on day X, you are probably going to start showing symptoms around day X+5 and shedding the virus on day X+4 to X+6. However, it's possibly you get exposed on day X, don't show symptoms for a long time, and start viral shedding 12 days before you start showing symptoms. Simina pointing out the long tail really helped me understand why these time periods seem so weirdly long!!!